test-support/src/main/java/org/testcontainers/testsupport/Flaky.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.testsupport;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * TODO: Javadocs
+ */
+@Retention(RUNTIME)
+@Target({METHOD, TYPE})
+public @interface Flaky {
+
+    String rationale();
```

Comment:
I would suggest adding `untilDate` field, so that we get back to it (read - it starts failing again) after some time

Reply:
Nice! Great idea.